verticadb-operator.yaml

```diff
@@ -66,6 +66,10 @@ update:
   github:
     identifier: vertica/vertica-kubernetes
     strip-prefix: v
+    use-tag: true
+  version-transform:
+    - match: '-\d+$'
+      replace: ''
```

Comment:
Should we be representing the `-0` in our version number? (So this version would be something like `25.2.0.0` or `25.2.0_p0`)